mne_bids/write.py

```diff
@@ -1649,7 +1692,8 @@ def write_raw_bids(raw, bids_path, events_data=None, event_id=None,
 
     # write to the scans.tsv file the output file written
     scan_relative_fpath = op.join(bids_path.datatype, bids_path.fpath.name)
-    _scans_tsv(raw, scan_relative_fpath, scans_path.fpath, overwrite)
+    _scans_tsv(raw, scan_relative_fpath,
+               scans_path.fpath, keep_source, overwrite)
```

Comment:
These are so many parameters now, can you please pass them as keywords to avoid accidents?